odoo/openupgrade/openupgrade_loading.py

```diff
@@ -157,6 +157,8 @@ def isrelated(model, k):
                     sorted([x[0] for x in v.selection]))
             else:
                 properties['selection_keys'] = 'function'
+        elif v.type == 'binary':
+            properties['attachment'] = str(getattr(v, "attachment", False))
```

Comment:
Then I don't get what do you want me to do? I let it as is? I think it's better to not need to propagate this value to the rest of the type of fields.